### PR TITLE
Change balance request flow

### PR DESCRIPTION
**Pull request by Namiii**
Implemented the new Request builder flow for Balance request

### CODE REVIEW

File: src/test/java/co/omise/live/LiveAccountRequestTest.java [hunk 1]
```diff
@@ -7,17 +7,13 @@
...
-    private final String STAGING_PKEY = "[YOUR_PKEY]";
-    private final String STAGING_SKEY = "[YOUR_SKEY]";
-    private final String EXPECTED_EMAIL = "YOUR_EMAIL";
+public class LiveAccountRequestTest extends BaseLiveTest{
 
     @Test
-    @Ignore("only hit when test on staging.")
+    @Ignore("only hit when test on live.")
     public void getAccount_success() throws Exception {
         // Give
```
Comment (nuxzero) [inline src/test/java/co/omise/live/LiveAccountRequestTest.java]: I think this comment no need. Could you delete it?

File: src/test/java/co/omise/live/LiveAccountRequestTest.java [hunk 2]
```diff
@@ -7,17 +7,13 @@
...
-        Client client = new Client(STAGING_PKEY, STAGING_SKEY);
+        Client client = new Client(getPublicKey(), getSecretKey());
 
         // When
```
Comment (nuxzero) [inline src/test/java/co/omise/live/LiveAccountRequestTest.java]: Could you delete it?

File: src/test/java/co/omise/live/LiveAccountRequestTest.java [hunk 3]
```diff
@@ -26,6 +22,6 @@ public void getAccount_success() throws Exception {
         System.out.println("Account retrieved: " + actualAccount.getEmail());
 
         // Then
```
Comment (nuxzero) [inline src/test/java/co/omise/live/LiveAccountRequestTest.java]: Could you delete it?

File: src/test/java/co/omise/requests/RequestTest.java [hunk 4]
```diff
@@ -31,4 +32,8 @@ protected void assertRequested(String method, String path, int code) {
         assertNotNull(response);
         assertEquals(code, response.code());
     }
+
+    protected Requester getTestRequester(){
```
Comment (nuxzero) [inline src/test/java/co/omise/requests/RequestTest.java]: Spacing 😄

**Comment by Namiii** [comment]
Done and done :D

**Pull request merged**